tests/test_reporters.py

```diff
@@ -531,3 +531,134 @@ def test_html_report_delta():
     </div>
   </body>
 </html>"""
+
+
+def test_html_report_delta__show_missing_False():
+    from pycobertura.reporters import HtmlReporterDelta
+
+    cobertura1 = make_cobertura('tests/dummy.source1/coverage.xml')
+    cobertura2 = make_cobertura('tests/dummy.source2/coverage.xml')
+
+    report_delta = HtmlReporterDelta(cobertura1, cobertura2, show_missing=False)
+    html_output = report_delta.generate()
+
+    print(html_output)
```

Comment:
Do we need this?

```suggestion
```

Reply:
Whoops! Debugging failure... couldn't get `-vv` to work properly to show my failed diff. Pushed and squashed.